conftest.py

```diff
@@ -168,17 +164,13 @@ def start_browser(
     slow_mo,
 ):
     browser_type = getattr(playwright, browser_name)
-    browser = browser_type.launch(
```

Comment:
It seems like a lot of these changes are related to reformatting, I'm surprised the linting checks hasn't picked these up.

If you run `ruff format` are these still here?